misaligned.c

```diff
@@ -10,12 +12,15 @@ int printColorMap() {
             printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
         }
     }
+    
+    TotalColorpairPrinted = ((i-1)*5)+(j-1);

```

Comment:
This statement is not related to the references manual? Why have it in the function?